apps/studio/components/to-be-cleaned/Storage/StorageSettings/StorageSettings.tsx

```diff
@@ -222,7 +232,7 @@ const StorageSettings = () => {
                         limit,
                         storageUnit
                       ).toLocaleString()} bytes. `}
-                    Maximum size in bytes of a file that can be uploaded is 50 GB (
+                    Maximum size in bytes of a file that can be uploaded is 500 GB (
```

Comment:
Should this text be dynamic (using `formattedMaxSizeBytes`) ?

Reply:
Yep - it was previously static but that was confusing - made it dynamic now